### ISSUE TITLE
IndentationWidth is not working properly

### ISSUE THREAD

**Issue by Ana06** [bbatsov/rubocop#4608]
IndentationWidth auto correction is not working properly and when running Rubocop again after the autocorrection it doesn't complain anymore, so the cop is not working properly.

--------

Lets take this code as example:

``` Ruby
word = if (count == 1 || count =~ /^1(\.0+)?$/)
  singular
else
  plural || singular.pluralize
end
```
## Expected behavior

when running `rubocop -a` you should get:

``` Ruby
word = if (count == 1 || count =~ /^1(\.0+)?$/)
         singular
       else
         plural || singular.pluralize
       end
```

## Actual behavior

when running `rubocop -a` you get:

``` Ruby
word = if (count == 1 || count =~ /^1(\.0+)?$/)
         singular
else
  plural || singular.pluralize
end
```
so the else clause is not corrected and if you run `rubocop` again no offenses are detected.
## RuboCop version

```
$ rubocop -V
0.48.1 (using Parser 2.4.0.0, running on ruby 2.2.6 x86_64-linux-gnu)
```

**Comment by Drenmi** [comment]
Do you have `ElseAlignment` and `EndAlignment` enabled? They are the respective cops that should investigate the `else` and `end` in this case.

**Comment by deivid-rodriguez** [comment]
I find this counter-intuitve, and I've been surprised exactly like @Ana06 is. Specially if you choose a `DisabledByDefault` approach to selectively introduce `rubocop` checks.

Why is this divided into three different cops? Does it make sense to enable one of them and not the others? I can see the `EndAlignment` being useful in other situations, but not the other two. The resulting styles (at least the one presented here) seems like something no one would use.

**Comment by Drenmi** [comment]
> I find this counter-intuitive [...]

Yes. Configuring this can be pretty hard. It confused me for a brief moment when reading this issue. Most of that difficulty, however, comes from having to translate our fuzzy thinking ("just indent the code the way I want") into some unambiguous specification.

We could maybe focus on having RuboCop help with this configuration experience, but I'm not sure how. It all needs to be explicated, regardless of whether that's done in a single cop or several.

> Why is this divided into three different cops?

Encapsulating all the possible options Ruby allows (probably hundreds of permutations) would lead to a truly gargantuan class. The existing ones are already pretty bad. 🙂 

`Style/IndentationWidth` checks the indentation of expressions relative to the related keyword. It is secondary to the cops checking the keywords themselves.

It gets even more complicated because some alignment 

> The resulting styles [...] seems like something no one would use.

If there's one thing I've learned, it is to not underestimate people's perceived need to create their own conventions ... 😅

**Comment by deivid-rodriguez** [comment]
Thanks for the answer @Drenmi.

I actually misunderstood the cop involved in this issue. I absolutely agree that `IndentationWidth` is a completely different beast. I thought the "buggy" cop involved here was `IfAlignment` or something like that.

I agree with everything you said, and share your doubts about how to make configuring this less surprising.

**Comment by jonas054** [comment]
Closing as the issue doesn't report any bugs. There are some shortcomings described, but I see no obvious way to make improvements.
